common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -662,6 +680,29 @@ public void execute(Runnable task) {
         }
     }
 
+    /**
+     * Returns the {@link Thread} that powers the {@link SingleThreadEventExecutor}.
+     */
+    public final ThreadDetails threadDetails() {
+        ThreadDetails state = threadDetails;
```

Comment:
Consider renaming the `state` variable. It doesn't seem very representative of how it is used.